src/fu-util-common.c

```diff
@@ -127,3 +127,10 @@ fu_util_is_interesting_device (FwupdDevice *dev)
 		return TRUE;
 	return FALSE;
 }
+
+gchar *
+fu_util_get_user_cache_path (const gchar *fn)
+{
+	g_autofree gchar *basename = g_path_get_basename (fn);
+	return g_build_filename (g_get_user_cache_dir (), "fwupd", basename, NULL);
```

Comment:
I do think it would be nice to make a symlink or move the files if the old path is found though too.

Reply:
I'm hesitant to rename, and a symlink kinda clutters things up. What about https://github.com/hughsie/fwupd/pull/700 ?